pyproject.toml

```diff
@@ -14,6 +14,8 @@ jsonschema = "^4.21.1"
 pydantic-settings = "^2.1.0"
 asyncpg = "^0.29.0"
 pandas = "^2.2.3"
+combat = "^0.3.3"
+conorm = "^1.2.0"
```

Comment:
Yes, it serves as a wrapper for conorm. Unfortunately, getmm is not yet implemented, though this could be addressed relatively easily. However, as we initially discussed, we can continue using conorm as start point